### DISCUSSION TITLE
Manual to use the API

### DISCUSSION THREAD

**Discussion by fabregasf** [bitcoin-core/secp256k1#1189] [Q&A]
Hey, how are the works? 
I have doubts about the project, first you have a doc to guide how to instantiate this API. I'm newbi in c/c++ programming, attending some classes to improve my knowlegment (structs, classes, typedefs, data structures ..)
I have a single project here with the secp256k1.h included? There is more header I should include to make it works great? I'm trying to get the instance as show below, in my code.

#include "secp256k1.h"
// lib do C++ 
secp256k1_context* ctx;
typedef SECP256K1_WARN_UNUSED_RESULT initialize;

Thanks in advance for all.

**Comment by elichai** [comment]
Hi :)
You can see https://github.com/bitcoin-core/secp256k1/pull/748 for usage examples,
don't forget you also need to compile this library and provide it to the linker.

If you have any specific errors / problems feel free to comment them here
